xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/v3/dto/account/FuturesBillsResponse.java

```diff
@@ -0,0 +1,44 @@
+package org.knowm.xchange.okcoin.v3.dto.account;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import java.math.BigDecimal;
+import java.util.Date;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
```

Comment:
could we use `@Value` instead of `@Data`?

Reply:
sure, @Value is a good option here, but i simply wanted to be consistent to other classes in this package